mappings/servers/twerion/metadata.json

```diff
@@ -0,0 +1,71 @@
+{
+    "id": "twerion",
+    "name": "Twerion Network",
+    "description": "A network focused on PvP",
+    "addresses": [
+        "twerion.net",
+        "twerion.de",
+        "twerion.com",
+        "twerion.eu",
+        "twerion.org"
+    ],
+    "primaryAddress": "twerion.net",
+    "minecraftVersions": [
+        "1.8.*"
+    ],
+    "primaryMinecraftVersion": "1.8.9",
+    "primaryColor": "#FFAA00",
+    "secondaryColor": "#FF8000",
+    "primaryLanguage": "en",
+    "languages": [
+        "en",
+        "de",
+        "ar",
+        "cn",
+        "cz",
+        "dk",
+        "es",
+        "fi",
+        "fr",
+        "gr",
+        "hi",
+        "hu",
+        "it",
+        "jp",
+        "lt",
+        "lv",
+        "nl",
+        "pl",
+        "pt",
+        "ro",
+        "ru",
+        "se",
+        "sk",
+        "sl",
+        "tr"
+    ],
```

Comment:
I completely missed this. The format for these languages do not match our required scheme. If you feel like we're missing some languages, let me know and I can consider getting them added to our scheme
![image](https://github.com/user-attachments/assets/d98991ac-2b31-43ce-bfb4-369e64eb78ed)